.redhat-ci.sh

```diff
@@ -7,13 +7,18 @@ mount --make-rshared /
 
 if [ -f /run/ostree-booted ]; then
     if [ ! -e /var/tmp/ostree-unlock-ovl.* ]; then
-        ostree admin unlock
+	# Until overlayfs and selinux get along, use remount
+	# instead of ostree admin unlock
+
+        #ostree admin unlock
+	mount -o remount,rw /usr
     fi
 else
     dnf install -y atomic python3-coverage
 fi
 
-systemctl start docker
+# Restarting docker helps with permissions related to above.
+systemctl restart docker
```

Comment:
Well it is a catch all.  In the case of the cloud image, docker is installed and not started.  So the restart here catches that.